VEHICLE/DeleteScriptVehicleGenerator.md

```diff
@@ -8,7 +8,7 @@ ns: VEHICLE
 void DELETE_SCRIPT_VEHICLE_GENERATOR(int vehicleGenerator);
 ```
 
+Removes a scripted vehicle generator.
```

Comment:
Any way of getting the integer value for this `vehicleGenerator`? Some example code would be nice 😅

Reply:
Probably missing the native `VEHICLE_GENERATOR_INDEX` that creates a vehicle generator. R* seems use `SET_SCRIPT_VEHICLE_GENERATOR` with it too.